src/System.DirectoryServices.AccountManagement/tests/PrincipalContextTests.cs

```diff
@@ -90,7 +93,8 @@ public void Ctor_ContextType_Name_Container_Options(ContextType contextType, str
             }
             else
             {
-                Assert.Throws<InvalidCastException>(() => context.ConnectedServer);
+                Assert.NotNull(context.ConnectedServer);
```

Comment:
I mean,
`Assert.Equal(Environment.MachineName, context.ConnectedServer)`

We have tests that make sure `Environment.MachineName` never returns null.

It's not important though